standard/variables.md

```diff
@@ -150,6 +150,10 @@ A local variable introduced by a *local_variable_declaration* is not automatical
 >
 > *end note*
 
+#### §discards-new-clause Discards
+
+A ***discard*** is a local variable that has no name, and can thus only be used once, at the point where it is introduced. A discard is not initially assigned, so it is always an error to access its value. A discard is introduced by a *declaration_expression* or *tuple_deconstruction* with the identifier `_`, and by use of the *simple_name* `_` when lookup of that name does not yield a result ().
+
```

Comment:
Somewhere here, we need to make sure that multiple discards are allowed in the same scope:

```csharp

_ = M1();  // Discard's type is int
_ = M2();  // Discard's type is string

int M1() => 1;
string M2() => "hello";
```

Reply:
I believe the treatment of discards in expressions.md (Simple Names and Declaration Expressions) takes care of that by ensuring they do not cause anything to be declared or added to the declaration space.